packages/components/src/molecules/Panels/ResizableColumnsPanel.tsx

```diff
@@ -25,9 +18,22 @@ const ResizableColumnsPanel: React.FC<ResizableColumnsPanelType> = (props) => {
   return (
     <ReflexContainer orientation="vertical" windowResizeAware style={{ height, width }}>
       {left && (
```

Comment:
we are checking this left on two separate times maybe we can bring them together like:
left && <Comp/>
left && <Another/>

left && <><Comp/><Another/></>

Reply:
For some reason, when using `<></>` the reflex splitter is not working, that is why we have two left conditions

Reply:
Please be mindful of changing Reflex code without **thoroughly** testing it. There are a lot of pitfalls that break things or ruin performance. I fixed some very subtle bugs that caused React components to be unmounted+recreated instead of rerendered all the time.

Reply:
What about React.Fragment

Reply:
Still not working...